client-java/controller/src/main/java/org/evomaster/client/java/controller/mongo/MongoHeuristicsCalculator.java

```diff
@@ -281,6 +283,50 @@ private double calculateDistanceForInvertedType(InvertedTypeOperation operation,
         return !Objects.equals(actualType, expectedType) ? 0.0 : MIN_DISTANCE_TO_TRUE_VALUE;
     }
 
+    private double calculateDistanceForNearSphere(NearSphereOperation operation, Object doc) {
+        String field = operation.getFieldName();
+        Object actualPoint = getValue(doc, field);
+
+        double x1 = Math.toRadians(operation.getLongitude());
+        double y1 = Math.toRadians(operation.getLatitude());
+        double x2;
+        double y2;
+
+        // GeoJsonPoint in document
+        if (isDocument(actualPoint) && getValue(actualPoint, "type").equals("Point") && getValue(actualPoint, "coordinates") instanceof List<?>) {
+
```

Comment:
hi @hghianni . you wrote "link added", but I do not see it here... did you push it?

Reply:
Hi @arcuri82, I added a link about GeoJsonPoint to the GsonJsonPointToOasConverter but I will add another link here about the case sensitivity.